Jenkinsfile

```diff
@@ -1,60 +0,0 @@
-#!/usr/bin/env groovy
-
-// See https://github.com/capralifecycle/jenkins-pipeline-library
-@Library('cals') _
-
-buildConfig([
-  slack: [
-    channel: '#cals-dev-info',
-    teamDomain: 'cals-capra',
-  ],
-]) {
-  dockerNode {
-    checkout scm
-
-    insideToolImage('node:18') {
-      stage('Install dependencies') {
-        sh 'npm ci'
-      }
-
-      stage('Lint') {
-        sh 'npm run lint'
-      }
-
-      stage('Prepare') {
-        sh 'npm run prepare'
-      }
-
-      stage('Tests') {
-        sh 'npm test'
-      }
-
-      analyzeSonarCloudForJs([
-        'sonar.organization': 'capralifecycle',
-        'sonar.projectKey': 'capralifecycle_cals-cli',
-      ])
-
-      // We only run semantic-release on the release branches,
-      // as we do not want credentials to be exposed to the job
-      // on other branches or in PRs.
-      //
-      // To have the correct version applied to the build we need
-      // to use a hook that is run during semantic-release execution.
-      // For this we use the 'prepack' hook. This also ensures
-      // that 'npm link' and alike builds the code, and we can
-      // trigger the same hook on other branches.
-
-      if (env.BRANCH_NAME ==~ /^(master|\d+\.(\d+|x)(\.x)?)$/) {
```

Comment:
Tok ikke med støtte for det alternative branchnavnet, regner med det dreide seg om f.eks denne branchen, [1.x](https://github.com/capralifecycle/cals-cli/tree/1.x), men den har ikke vært oppdatert på årevis